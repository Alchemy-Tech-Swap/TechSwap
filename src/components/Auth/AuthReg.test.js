import AuthReg from './AuthReg';
import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

test('Should render AuthReg.js', async () => {
  const { container } = render(
    <MemoryRouter>
      <AuthReg />
    </MemoryRouter>
  );

  await screen.findByText('Email:');

  expect(container).toMatchSnapshot();
});
