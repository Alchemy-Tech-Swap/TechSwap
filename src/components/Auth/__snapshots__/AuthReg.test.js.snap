// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render AuthReg.js 1`] = `
<div>
  <form
    class="form"
  >
    <h3 />
    <div
      class="form-controls"
    >
      <label>
        Email:
      </label>
      <input
        placeholder="Enter Valid Email Address"
        type="email"
        value=""
      />
    </div>
    <div
      class="form-controls"
    >
      <label>
        Password:
      </label>
      <input
        placeholder="Minimum 6 digits req."
        type="password"
        value=""
      />
    </div>
    <div
      class="form-controls"
    >
      <label>
        Name:
      </label>
      <input
        placeholder="Name"
        type="text"
        value=""
      />
    </div>
    <div
      class="form-controls"
    >
      <label>
        Slack:
      </label>
      <input
        placeholder="Slack User ID"
        type="text"
        value=""
      />
    </div>
    <div
      class="form-controls"
    >
      <label>
        LinkedIn
      </label>
      <input
        placeholder="LinkedIn URL"
        type="text"
        value=""
      />
    </div>
    <input
      class="submit"
      type="submit"
    />
  </form>
</div>
`;
